File: packages/core/src/models/UsageSnapshot.ts
/**
 * Снимок использования провайдера
 */
export type UsageStatus = "ok" | "warning" | "error";

export interface UsageSnapshot {
  providerId: string;
  used: number;
  limit: number;
  unit: string;
  timestamp: number;
  status: UsageStatus;
  /** Отображаемое имя провайдера */
  displayName?: string;
}

File: apps/android/app/src/main/java/com/ailimits/dashboard/data/UsageSnapshot.kt
package com.ailimits.dashboard.data

data class UsageSnapshot(
    val providerId: String,
    val used: Double,
    val limit: Double,
    val unit: String,
    val timestamp: Long,
    val status: UsageStatus,
    val displayName: String? = null,
)

enum class UsageStatus { OK, WARNING, ERROR }
